.cargo/config.toml

```diff
@@ -0,0 +1,24 @@
+# Rename this file to `config.toml` to enable "fast build" configuration. Please read the notes below.
```

Comment:
I'm guessing nightly is the more useful build during development, and we can move to stable rust for production builds?

Reply:
Generally, that is the plan, yes. Much faster builds on nightly. Nightly builds are sometimes faster and smaller, too. It's not generally a bad idea to use it imo, honestly, but also, things could break in unexpected ways. If we ever suspect that's the case, building with `cargo +stable run` could help to investigate that.